pulsar-client-api/src/main/java/org/apache/pulsar/client/api/ClientBuilder.java

```diff
@@ -337,7 +337,7 @@ ClientBuilder authentication(String authPluginClassName, Map<String, String> aut
      * Set keep alive interval in seconds for each client-broker-connection. <i>(default: 30)</i>.
      *
      * @param keepAliveIntervalSeconds
-     * @param unit time unit for {@code statsInterval}
+     * @param unit the time unit in which the keepAliveIntervalSeconds is defined
      * @return the client builder instance
      */
     ClientBuilder keepAliveInterval(int keepAliveIntervalSeconds, TimeUnit unit);
```

Comment:
we need remove seconds from `keepAliveIntervalSeconds` and also update the javadoc.

Reply:
ok